inst/settings/resultsDataModelSpecification.csv

```diff
@@ -228,8 +228,8 @@ visit_context,visit_concept_id,int,Yes,Yes,No,Yes,No,No,No
 visit_context,visit_context,varchar,Yes,Yes,No,Yes,No,No,No
 visit_context,subjects,float,Yes,No,No,Yes,Yes,No,No
 visit_context,database_id,varchar,Yes,Yes,No,Yes,No,No,No
-vocabulary,vocabulary_id,varchar(50),Yes,Yes,No,Yes,No,Yes,No
+vocabulary,vocabulary_id,varchar(50),Yes,No,No,Yes,No,Yes,No
```

Comment:
This is a standard vocabulary table - shipped by OMOP. This change is making me uncomfortable.

Reply:
I agree, but I can't think of any other way to support the VA